frontend/src/components/SignUp/page.tsx

```diff
@@ -46,12 +43,12 @@ export default function SignUpPage() {
         last_name: last_name,
       });
 
-      navigate("/dashboard");
+      navigate("/dashboard?uid=" + userCredentials.user.uid);
```

Comment:
i thought we wont be passing id in the params anymore? Also its ?id not ?uid

Reply:
based off #48

Reply:
oh oopsies, for now we leave as is, but yeah ill update it to be ?id

Reply:
i read #48 and it does seem very easy to grab id from the auth context provider.

It wont require any unnecessary imports of firebase package to components right? since its a context.

Reply:
yes, i was just thinking when that refactor occurs that it will happen in a singular commit